utils/roles.py

```diff
@@ -0,0 +1,71 @@
+import discord
+from bot_globals import MILESTONE_ROLES, STREAK_ROLES, logger, client
+
+async def give_user_streak_role(user: discord.User, guild_id: int, streak: int) -> None:
+    logger.info(f"file: utils/stats.py ~ give_user_streak_role ~ run - {user.id} - {streak}")
```

Comment:
you can't use f strings with logger. You have to use %s syntax

Reply:
Odd in my environment 3.11.3 I can use fStrings w/ logger

Output:
`2023-08-16 20:45:00,684 - INFO - file: utils/stats.py ~ give_user_streak_role ~ run - 199662959964848128 - 32`

Happy / fine for this refactor though to be internally consistent